mod/dfrn_poll.php

```diff
@@ -499,20 +498,20 @@ function dfrn_poll_content(App $a)
 
 			// URL reply
 			if ($dfrn_version < 2.2) {
-				$s = Network::fetchUrl($r[0]['poll']
-					. '?dfrn_id=' . $encrypted_id
-					. '&type=profile-check'
-					. '&dfrn_version=' . DFRN_PROTOCOL_VERSION
-					. '&challenge=' . $challenge
-					. '&sec=' . $sec
+				$s = DI::httpRequest()->fetch($r[0]['poll']
+				                              . '?dfrn_id=' . $encrypted_id
```

Comment:
Just some quick note: Is the indention correct?

Reply:
Apparently so, to align with `$r[0]['poll']`. I was in the middle of a massively conflicted rebase with 18 steps, so I didn't touch the parts that weren't conflicting!